src/plugins/PluginsContext.ts

```diff
@@ -0,0 +1,24 @@
+import {createContext} from 'react';
+
+export interface PluginData {
+  getComponent: (props?: any) => any;
+  order?: number;
+}
+
+export type PluginItems = Record<string, PluginData[]>;
```

Comment:
I don't think that these are `PluginItems`, but I don't have any good name for that; I'm not sure though if the name is even needed